core/templates/pages/donate-page/donate-page.module.ts

```diff
@@ -19,12 +19,21 @@
 import 'core-js/es7/reflect';
 import 'zone.js';
 
+angular.module('oppia', [
+  'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
+  'ngAudio', require('angular-cookies'), 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
+  'ngSanitize', 'ngTouch', 'pascalprecht.translate',
+  'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+]);
```

Comment:
Could you please remove some of the dependencies here? Most of them are not needed on the donate page, you could inspire in https://github.com/oppia/oppia/blob/develop/core/templates/pages/pending-account-deletion-page/pending-account-deletion-page.module.ts#L75 but the donate page needs to be check that everything works.

Reply:
Done